src/rsz/src/BaseMove.cc

```diff
@@ -573,8 +573,16 @@ bool BaseMove::hasPort(const Net* net)
     return false;
   }
 
-  dbNet* db_net = db_network_->staToDb(net);
-  return !db_net->getBTerms().empty();
+  odb::dbNet* db_net = nullptr;
```

Comment:
It feels like this method should belong to dbNetwork as it has no rsz related code in it and could be useful in other contexts.